daemon/dpudaemon.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"context"
+	"net"
+
+	"github.com/go-logr/logr"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	"google.golang.org/grpc"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type DpuDaemon struct {
+	pb.UnimplementedBridgePortServiceServer
+	dev bool
+	vsp VendorPlugin
```

Comment:
Same comment here about `vsp *VendorPlugin`

Reply:
Because `NewDummyPlugin(mode string) *DummyPlugin` returns a pointer to a vsp object. I assume all VSPs being stored would be pointer objects.